benchmarks/reframe_config.py

```diff
@@ -752,6 +752,8 @@ def spack_root_to_path():
                         '%(check_environ)s|'
                         '%(check_extra_resources)s|'
                         '%(check_env_vars)s|'
+                        '%(check_compiler_name)s|'
+                        '%(check_compiler_version)s|'
```

Comment:
Don't you also want the variant fished out of the `spack_spec`?

Reply:
Right now, seems like not needed anymore since `tags` from ReFrame has the programming model variant, and compiler name alongside with version is provided with this PR. In the new Spack package we have made some changes to make sure each programming model has its own dedicated flags. So, it is tough to have a one solution fits all for the rest of the variants for Babelstream and also for other apps too since naming conventions are all different.

Reply:
Then maybe we should export the whole variant? And users of each app can choose to use it and unpack it or not.